Gemfile

```diff
@@ -2,3 +2,6 @@ source 'https://rubygems.org'
 
 # jekyll-sitemap included in github-pages deps
 gem 'github-pages', '>= 22'
+
+# Added at 2018-05-23 15:03:09 -0400 by isaacmurchie:
+gem "rawler", "~> 0.1.8"
```

Comment:
is it not `crawler`?

Reply:
No (https://github.com/oscardelben/rawler). But it's moot because this isn't needed.